src/views/MissionPlanningView.vue

```diff
@@ -202,16 +264,22 @@
                 />
               </template>
             </v-tooltip>
+            <v-divider vertical />
+            <v-tooltip location="top" text="Mission Settings">
+              <template #activator="{ props }">
+                <v-btn
+                  v-bind="props"
+                  icon="mdi-cog"
+                  variant="text"
+                  size="24"
+                  class="text-[12px] mx-3 mt-[2px] mb-[1px]"
+                  @click="handleOpenMissionSettings"
+                />
+              </template>
+            </v-tooltip>
           </div>
         </div>
         <v-divider v-if="isCreatingSimplePath || isCreatingSurvey" class="my-2" />
-        <button
-          v-if="missionStore.currentPlanningWaypoints.length > 0"
-          class="h-auto py-1 px-1 m-2 mt-2 text-sm rounded-md elevation-1 bg-[#FFFFFF11] hover:bg-[#FFFFFF22] transition-colors duration-200"
-          @click="clearCurrentMission"
-        >
-          <p>CLEAR CURRENT MISSION</p>
-        </button>
```

Comment:
This was just added in the previous commit, and is being removed in the following commit.
14 extra lines to be reviewed.